### PR TITLE
Added PowerScale v2.8.0 config

### PR DESCRIPTION
**Pull request by chimanjain**
# Description

Added PowerScale v2.8.0 config

# GitHub Issues

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/885 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Ran Sanity test and unit test

### CODE REVIEW

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by shefali-malhotra — DISMISSED** [review]
LGTM

File: samples/storage_csm_powerscale_v280.yaml [hunk 1]
```diff
@@ -395,3 +433,42 @@ spec:
...
             - name: "COLLECTOR_ADDRESS"
               value: "otel-collector:55680"
+    - name: resiliency
+      # enabled: Enable/Disable Resiliency feature
+      # Allowed values:
+      #   true: enable Resiliency feature(deploy podmon sidecar)
+      #   false: disable Resiliency feature(do not deploy podmon sidecar)
+      # Default value: false
+      enabled: false
+      configVersion: v1.6.0
```
Comment (adarsh-dell) [inline samples/storage_csm_powerscale_v280.yaml:443]: Hi @chimanjain Can you please confirm the configVersion of resiliency module?
I think it should be v1.7.0 i.e. latest one that we will be releasing in this quarter.

File: operatorconfig/moduleconfig/common/version-values.yaml [hunk 2]
```diff
@@ -14,6 +10,11 @@ powerscale:
       replication: "v1.5.0"
       observability: "v1.5.0"
       resiliency: "v1.6.0"
+  v2.8.0:
+      authorization: "v1.7.0"
+      replication: "v1.5.0"
+      observability: "v1.5.0"
+      resiliency: "v1.6.0"
```
Comment (adarsh-dell) [inline operatorconfig/moduleconfig/common/version-values.yaml:17]: resiliency: "v1.6.0" -> resiliency: "v1.7.0" ? Correct the version here too, like CSI-PowerStore.

File: samples/storage_csm_powerscale_v280.yaml (hunk 1 above)
Comment (chimanjain) [reply samples/storage_csm_powerscale_v280.yaml:443]: Done

File: operatorconfig/moduleconfig/common/version-values.yaml (hunk 2 above)
Comment (chimanjain) [reply operatorconfig/moduleconfig/common/version-values.yaml]: Done

**Comment by chimanjain** [comment]
PTAL

**Review by shefali-malhotra — DISMISSED** [review]
LGTM

**Review by adarsh-dell — APPROVED** [review]

File: tests/config/driverconfig/powerscale/v2.8.0/controller.yaml [hunk 3]
```diff
@@ -187,14 +197,25 @@ spec:
...
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
           volumeMounts:
             - name: socket-dir
               mountPath: /var/run/csi
         - name: snapshotter
-          image: registry.k8s.io/sig-storage/csi-snapshotter:v6.1.0
+          image: registry.k8s.io/sig-storage/csi-snapshotter:v6.2.1
```
[the remark this answers is not in the document]
Comment (chimanjain) [reply tests/config/driverconfig/powerscale/v2.8.0/controller.yaml]: Fixed

**Review by panigs7 — APPROVED** [review]

**Review by rensyct — APPROVED** [review]

**Pull request merged**